### PR TITLE
[7] Develop Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by michaelolubajo**
## Task: `Developed contactUs.html page
- `Created page, created form and linked to product page`
`

Screenshot_2020-07-27 Contact Us Page

### CODE REVIEW

**Comment by adaugochi** [comment]
Hello @michaelolubajo Welldone

But, task 7 is just for creating contact form page. You already created a product page in task 6

**Comment by michaelolubajo** [comment]
Thanks.

I created the product page in task 6 but like the other guy mentioned I
couldn't find the folder in my local and so I saved another and added task
7. I just want that merged so I can add task 8 immediately. It's already
done.

Thanks again.

Regards,
Michael.

File: submissions/michaelolubajo/htmlForms/contactUs.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	</div>
+	<div class="topnav">
+  		<a class="active" href="#home">Home</a>
+ 		<a href="../html/productpage.html">Product</a>
+  		<a href="contactus.html">Contact Us</a>
+  		<a href="signup.html">Sign Up</a>
+	</div>
+
+	<h1>Contact Us</h1>
+	<form action="#" method="post" enctype="multipart/form-data">
```
Comment (ToluAkin) [inline submissions/michaelolubajo/htmlForms/contactUs.html:20]: Remove the attributes `action`, `method`, `enctype` in the form tag.

File: submissions/michaelolubajo/htmlForms/contactUs.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+				<input type="email" name="email" id="email" placeholder="Enter your email address"><br><br>
+				<label for="subject">Subject</label><br>
+				<input type="text" name="subject" id="subject" placeholder="Enter subject of message"><br><br>
+				<label>Message</label><br>
+				<textarea name="message" id="message" cols="50" rows="10" placeholder="Enter your message here"></textarea><br><br>
+				<button type="submit">Submit</button>
+
+		</fieldset>
+
+
```
Comment (ToluAkin) [inline submissions/michaelolubajo/htmlForms/contactUs.html:36]: Remove extra space

**Pull request merged**